scripts/vmbuild.rb

```diff
@@ -170,21 +170,24 @@ def verify_run(output)
   end
 end
 
-# Only update the symlink for a nightly
+# Only update the latest symlink for a nightly
 unless build_label == "test"
-  link = stream_directory.join("latest")
+  symlink_name = "latest"
+  link = stream_directory.join(symlink_name)
   if File.exist?(link)
     raise "#{link} is not a symlink!" unless File.symlink?(link)
     result = FileUtils.rm(link, :verbose => true)
     $log.info("Deleted symlink: #{result}")
   end
 
-  result = FileUtils.ln_s(destination_directory, link, :verbose => true)
-  $log.info("Created symlink: #{result}")
+  Dir.chdir(stream_directory) do
+    result = FileUtils.ln_s(directory_name, symlink_name, :verbose => true)
```

Comment:
/mnt/builds happens to be the same for all build machines (except virtlab31 which has it mounted on /home/appliances/cfme). so build VMs will put their builds in their relative directories, i.e. /mnt/builds/kegerator/, /mnt/builds/aab-brewery7, i.e. the /build/fileshare for that particular build machine.  so /mnt/builds/upstream/latest does not apply here, really is /mnt/builds/kegerator/upstream/latest.    Nice thing about the changes in this PR, it does not matter how/where things are mounted, as as long as its mounted, you can see/traverse the latest links from any VM build machine. even if it's /home/appliances/cfme/ or anything else for that matter.  I hope this makes sense.